client/MentorPanel/src/redux/store.js

```diff
@@ -54,6 +58,12 @@ const reducer = (state = initialState, action) => {
         faved_mentor: state.faved_mentor === action.id ? '' : action.id
       }
 
+    case MENTOR_NEXT:
+      return {
+        ...state,
+        next_mentor: action.mentor,
+      }
+
     case QUESTION_SENT:
       return {
```

Comment:
super small thing, but you can return state here without copying since you're not modifying it